docs/docs/references/the-sql-runner.mdx

```diff
@@ -0,0 +1,37 @@
+import AccessSQLRunner from './assets/access-sql-runner.jpg';
+import SQLRunnerComponents from './assets/sql-runner-components.jpg';
+import ShareSQLRunnerChart from './assets/share-sql-runner-chart.jpg';
+
+# The SQL Runner
```

Comment:
Do we want to add a note about roles and permissions for SQL runner?

Reply:
yep, good shout